FESS-GUI/setpassworddialog.cpp

```diff
@@ -13,6 +14,7 @@ SetPasswordDialog::SetPasswordDialog(QWidget *parent) :
     ui(new Ui::SetPasswordDialog)
 {
     ui->setupUi(this);
+    qsrand(time(NULL));
```

Comment:
This causes a compile error on my machine because time is not declared. Include ctime if you want to do this.